release/release.md

```diff
@@ -103,20 +103,30 @@ $ git pull
    $./gen_versions_txt.sh ${BRANCH}
    $PUSH=1 OBS_SUBPROJECT="releases:$(uname -m):${BRANCH}" ./build_from_docker.sh ${NEW_VERSION}
    ```
+
+6. Test packages
```

Comment:
markdown hint - if you label all numbered items `1`, then the markdown formatter will auto-increment them for you when it renders - which makes it much easier for you later to add/remove/reshuffle your items.

Reply:
I've tried in the past in this document and failed, seems that information between items confuses the render and is not automatically updated.